### ISSUE TITLE
[Windows] build doesn't output "app-release-signed.apk" - no errors

### ISSUE THREAD

**Issue by jabza** [GoogleChromeLabs/bubblewrap#190]
**Describe the bug**
Running bubblewrap build --skipPwaValidation appears to successfully complete, but does not
generate a signed apk, only a 'app-release-unsigned-aligned.apk' exists in the directory.

**To Reproduce**
1. init a new project with no keystore
2. run bubblewrap build with --skipPwaValidation
3. check folder for signed apk

**Expected behavior**
A signed apk named app-release-signed.apk is output or an error is given.

**Desktop (please complete the following information):**
 - OS: Windows 10

I'm using --skipPwaValidation as my PWA pass all Installable tests on Chrome Lighthouse.

**Comment by jabza** [comment]
After futher investigation, it is because find_java.bat does not exist in the android cli tools.

I fixed this problem by hard coding my java .exe in the apksigner.bat file:

`rem Check we have a valid Java.exe in the path.
set java_exe=C:\Program Files\AdoptOpenJDK\jdk-8.0.252.09-hotspot\bin\java.exe
if exist    "%~dp0..\tools\lib\find_java.bat" call    "%~dp0..\tools\lib\find_java.bat"
if exist "%~dp0..\..\tools\lib\find_java.bat" call "%~dp0..\..\tools\lib\find_java.bat"
if not defined java_exe goto :EOF`

any ideas why find_java.bat does not exist? For the record, I build-tools 29.0.2 and 28.0.3. find_java.bat was missing in both.

**Comment by andreban** [comment]
This is an issue with the Android command-line tools, so not much we can do in Bubblewrap: https://issuetracker.google.com/issues/150888434

Older versions of the Android SDK tools should work: https://dl.google.com/android/repository/sdk-tools-windows-4333796.zip

**Comment by andreban** [comment]
A possible workaround is to run `apksigner.jar` directly, without invoking the `.bat`:

```
 java -Xmx1024M -Xss1m -jar "D:\path-to\android-cli-tools\build-tools\29.0.2\lib\apksigner.jar"  sign --ks ./android.keystore --ks-key-alias android --ks-pass pass:android --key-pass pass:android --out ./app-release-signed.apk ./app-release-unsigned-aligned.apk
```

**Comment by andreban** [comment]
#191 implements a workaround for this

**Comment by andreban** [comment]
#191 adds a workaround for this. It's published on v1.3.1. Let me know if this works.
